shared/CHANGELOG.md

```diff
@@ -7,9 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.53] - 2023-04-18
+
+- Added a new `Session` protobuf
+- Added a `GetActor` function to the Persistence module interface
+- Added a `GetSession` function to the Utility module interface
+
 ## [0.0.0.52] - 2023-04-17
 
-- Removed *temporary* `shared/p2p` package; consolidated into `p2p`
+- Removed _temporary_ `shared/p2p` package; consolidated into `p2p`
```

Comment:
nit: 😲 should  I have used `__` instead of `**`? Is this documented anywhere? Do you have opinions about modifying changelog history?

Reply:
I don't have a preference, it was just an automatic thing vscode did when I saved. Ideally we should have shared markdown lint configs: https://github.com/pokt-network/pocket/issues/257

No strings opinions about changing changelog history other than we shouldn't do it. Formatting is a bit less important so definitely a blurry line.